src/encoded/static/scss/encoded/modules/_cart.scss

```diff
@@ -1660,9 +1671,229 @@ $file-view-checkbox-size: 14px;
     margin-bottom: 10px;
 }
 
+.search-results-description {
+    word-break: normal;
+}
+
 .remove-multiple-control {
     .remove-multiple-control__note {
         font-size: 0.9rem;
         color: #606060;
     }
 }
+
+.cart-title-status {
+    display: flex;
+}
+
+// Cart description as it appears on the cart-view page.
+.cart-view-description {
+    margin: 10px 0;
+
+    @media screen and (min-width: $screen-md-min) {
+        width: 70%;
+    }
+
+    @media screen and (min-width: $screen-lg-min) {
+        width: 60%;
+    }
+
+    @media screen and (min-width: $screen-xl-min) {
+        width: 50%;
+    }
+
+    .cart-view-description__display {
+        &.cart-description__display--active {
+            padding: 5px;
+            border: 1px solid #e0e0e0;
+        }
+    }
+
+    .cart-view-description__edit {
+        float: right;
+    }
+
+    .cart-view-description__edit-button {
+        display: block;
+        padding: 2px;
+        border: none;
+        background: none;
+
+        .svg-icon-edit {
+            display: block;
+            width: 22px;
+            height: 22px;
+            fill: #808080;
+
+            @media screen and (min-width: $screen-sm-min) {
+                width: 14px;
+                height: 14px;
+            }
+        }
+    }
+}
+
+// Markdow-formatted cart description.
+.cart-description {
+    word-break: normal;
+
+    // Description text styles
+    h1, h2, h3, h4, h5, h6 {
+        font-family: $site-font;
+        border-bottom: none;
+
+        &:first-child {
+            margin-top: 0;
+        }
+
+        + p {
+            margin-top: 0;
+        }
+    }
+
+    h1 {
+        margin: 1.3rem 0 0.3rem;
+        font-size: 1.3rem;
+        font-weight: 700;
+    }
+
+    h2 {
+        margin: 1.2rem 0 0.2rem;
+        font-size: 1.2rem;
+        font-weight: 700;
+    }
+
+    h3 {
+        margin: 1.2rem 0 0.2rem;
+        font-size: 1.1rem;
+        font-weight: 700;
+    }
+
+    h4 {
+        margin: 1.1rem 0 0.2rem;
+        font-size: 1.1rem;
+        font-weight: 600;
+    }
+
+    h5 {
+        margin: 1.1rem 0 0.1rem;
+        font-size: 1rem;
+        font-weight: 600;
+    }
+
+    h6 {
+        font-size: 1rem;
+        margin: 1.1rem 0 0.1rem;
+        font-weight: 500;
+    }
+
+    p {
+        margin: 0.9rem 0;
+
+        &:first-child {
+            margin-top: 0;
+        }
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+
+    ul, ol {
+        margin: 0.3em;
```

Comment:
should we be using "em"?

Reply:
I copied this from a post with a URL I forgot to include that was for resetting list styles. So, I’m not sure in this case; I presume this had been tested, and I tested it as well. Not sure if there are cases this wouldn’t work.